src/moscot/base/output.py

```diff
@@ -245,20 +244,13 @@ def sparsify(
         else:
             raise NotImplementedError(mode)
 
-        if n < m:
-            k = n
-            func = self.push
-            fn_stack = sp.vstack
-        else:
-            k = m
-            func = self.pull
-            fn_stack = sp.hstack
+        k, func, fn_stack = (n, self.push, sp.vstack) if n < m else (m, self.pull, sp.hstack)
         tmaps_sparse: List[sp.csr_matrix] = []
         for batch in range(0, k, batch_size):
             x = np.eye(k, min(batch_size, k - batch), -(min(batch, k)))
             res = func(x, scale_by_marginals=False)
             res[res < thr] = 0
-            tmaps_sparse.append(sp.csr_matrix(res.T if func == self.push else res))
+            tmaps_sparse.append(sp.csr_matrix(res.T if func is self.push else res))
```

Comment:
@michalk8 this does not work, so will revert back to `==`

Reply:
This should work, maybe you tried with `autoreload`?

Reply:
I tried it again, and locally tests fail again. I just now pushed another version with `is`, let's see whether CI fails.

Reply:
It might be because `push` is decorated... Maybe `if n < m` is better in this case.